js/interface-lists.js

```diff
@@ -457,8 +457,20 @@ function attahObservers() {
         if (widgetData.filterOptions.length) {
           var filterError = [];
           var filterFieldValues = [];
+          var nonValidatedLogicOptions = [
```

Comment:
The variable name doesn't make sense.

1. It's `invalid`, not `nonValid`.
2. I don't understand why these are invalid. They look like valid logic options to me. This shows that the variable name is inappropriately named.

Reply:
@tonytlwu let me tell you what I mean here please, the logic of this check is to exclude checking of the Value type field when it is hidden and inactive, this array is responsible for each type of logic in which this field is hidden

Reply:
```suggestion
          var logicOptionsWithoutValues = [
```